src/tests/functional/shared_test_classes/src/single_layer/experimental_detectron_roifeatureextractor.cpp

```diff
@@ -46,6 +47,9 @@ std::string ExperimentalDetectronROIFeatureExtractorLayerTest::getTestCaseName(
 }
 
 void ExperimentalDetectronROIFeatureExtractorLayerTest::SetUp() {
+    // w/a for myriad (cann't store 2 caches simultaneously)
```

Comment:
Minor: Could you please add the ticket number?>

Reply:
Done